integration/build.gradle.kts

```diff
@@ -0,0 +1,53 @@
+import org.jetbrains.kotlin.gradle.dsl.*
+
+plugins {
+    kotlin("jvm")
+}
+
+repositories {
+    mavenCentral()
+}
+
+evaluationDependsOn(":kotlinx-benchmark-runtime")
+
+val runtime get() = project(":kotlinx-benchmark-runtime")
+val plugin get() = gradle.includedBuild("plugin")
+
+val AbstractArchiveTask.archiveFilePath get() = archiveFile.get().asFile.path
+
+fun artifactsTask(artifact: String) = runtime.tasks.getByName<AbstractArchiveTask>("${artifact}Jar")
+fun artifactsTaskNativeKlibs() = runtime.tasks.getByName("compileKotlinNative")
```

Comment:
The Gradle task `compileKotlinNative` only exists when it is run in the IDEA environment because of the `kotlinx.team.infra` plugin, which merges the same tasks of all native targets under "native" wording.
That is `compileKotlinLinuxX64`, `compileKotlinMingwX64`, and `compileKotlinMacosX64` are unified to `compileKotlinNative`, and running that task delegates to the host task.

However, Gradle won't find `compileKotlinNative` if you run it from the terminal. You can workaround this with smth  like:
```
val nativeHost = if (System.getProperty("idea.active") == "true") {
    "native"
} else when {
    org.jetbrains.kotlin.konan.target.HostManager.hostIsLinux -> "linuxX64"
    org.jetbrains.kotlin.konan.target.HostManager.hostIsMingw -> "mingwX64"
    org.jetbrains.kotlin.konan.target.HostManager.hostIsMac -> "macosX64"
    else -> error("Unknown host: ${org.jetbrains.kotlin.konan.target.HostManager.host}")
}
```

Reply:
Please rebase on master first and then use `getSystemProperty("idea.active")` instead of `System.getProperty`.
Here is the reason why: https://github.com/Kotlin/kotlinx-benchmark/pull/40

Reply:
Ok, will do it
I was thinking about it, that it could not work on CI:)

Reply:
it's internal in plugin + we don't have plugin at classpath there, I was just copied this functions. Not sure it's best way, maybe you can advise something else